service/worker/indexer/esProcessor.go

```diff
@@ -237,7 +237,10 @@ func (p *esProcessorImpl) hashFn(key interface{}) uint32 {
 		return 0
 	}
 	numOfShards := p.config.IndexerConcurrency()
-	return uint32(common.WorkflowIDToHistoryShard(id, numOfShards))
+
```

Comment:
There was no need to use WorkflowIDToHistoryShard as the hashing function.  All this logic needed is a hashing function which takes in a string and computes range hashKey which is used to route the call to worker go routine processing ES messages.